VeterinaryVladimir/Clinica.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VeterinaryVladimir
+{
+    class Clinica 
+    {
+        private List<CartAnimal> animals;
+
+        public Clinica()
+        {
+            animals = new List<CartAnimal>();
```

Comment:
it can be created just on declaration. C'tor usually used for more complicated initialization.